guides/common/modules/con_installing-and-configuring-the-team-advisor-engine-in-a-disconnected-environment.adoc

```diff
@@ -0,0 +1,18 @@
+[id="installing-and-configuring-the-team-advisor-engine-in-a-disconnected-environment"]
+= Installing and configuring the {Team} advisor engine
```

Comment:
This renders as "the Red Hat advisor engine." That's not what we want, is it?

Reply:
I can remove it if we don't want this.

Reply:
Per @ehelms [comment](https://github.com/theforeman/foreman-documentation/pull/3785#discussion_r2070206953) above:

```
For Foreman I think this should just read "The advisor engine" and for Satellite "The Insights in Satellite advisor engine"
```

Should change so every occurrence renders that way.